activateConnection.py

```diff
@@ -0,0 +1,39 @@
+"""
+Activate a connection by name
+"""
+
+import NetworkManager
+import sys
+
+# Find the connection
+name = sys.argv[1]
+connections = NetworkManager.Settings.ListConnections()
+connections = dict([(x.GetSettings()['connection']['id'], x) for x in connections])
+conn = connections[name]
```

Comment:
Yeah, however both connections I enable will always be present and this is a quick impl. that will be removed once the rust Network Manager binary is released is ~2 weeks